src/collectd.conf.pod

```diff
@@ -6625,6 +6633,29 @@ their configuration can be found in L<types.db(5)>.
 
 This optional setting sets the type instance to use.
 
+=item B<LatencyPercentile> I<Percent>
+
+Calculate and dispatch the configured percentile, i.e. compute the latency, so
+that I<Percent> of all matched latency values are smaller than or equal to the
+computed latency.
+
+Different percentiles can be calculated by setting this option several times.
+
+=item B<LatencyPercentileType> I<Type>
```

Comment:
I think that we know quite precisely that we're reporting a latency and as such should use the `latency` type. Allowing users to use a different *type* has next to no benefit and will result in more users seeking help to set this up.

With `LatencyPercentileType` removed, there is no need to have `LatencyRateType` – the rates should be reported using the general `Type` option.

I hope that this simplifies the config and reduces the number of support requests by users significantly.